.gitignore

```diff
@@ -26,6 +26,9 @@ test/EndToEnd/nuget.exe
 # remove ilmerge log
 mergelog.txt
 
+# MSBuild binlog
+*.binlog
```

Comment:
👏 

I should've done this long time ago haha. 

This should part of the default C# ignore files by now :D